balancer/xds/xds_client.go

```diff
@@ -68,13 +69,17 @@ type client struct {
 	loseContact  func(ctx context.Context)
 	cleanup      func()
 	backoff      backoff.Strategy
+	loadStore    lrs.Store
 
 	mu sync.Mutex
 	cc *grpc.ClientConn
 }
 
 func (c *client) run() {
 	c.dial()
+	if c.loadStore != nil {
+		go c.loadStore.ReportTo(c.ctx, c.cc)
```

Comment:
If there are a couple of xds clients alive (balancer name change or child policy change), then a potential issue is that we may miss some load reports due to the reset of loadstore drops field whenever buildStats is called. 

It's better to start load report when the first ADS response is received.

Reply:
Good point. Fixed!